src/main/java/Duke.java

```diff
@@ -1,10 +1,80 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("------------------------------------------------");
+        System.out.println("Hello, I am Chris");
+        System.out.println("------------------------------------------------");
+        boolean done = false;
+        Scanner in = new Scanner(System.in);
+        Task[] taskList = new Task[100];
+        int taskCount = 0;
+        while (!done) {
+            System.out.println("How can I help you today?");
+            System.out.println("1) Task List");
+            System.out.println("2) Add tasks");
+            System.out.println("3) Remove tasks");
+            System.out.println("4) Mark Tasks");
+            System.out.println("5) Quit");
+            String line = in.nextLine();
+            switch (line) {
+                case "1":
+                    if (taskCount == 0) {
+                        System.out.println("List is empty.");
+                    } else {
+                        for (int i = 0; i < taskCount; i++) {
+                            System.out.print(i + 1);
+                            System.out.print(". ");
+                            System.out.println(taskList[i]);
+                        }
+                    }
+                    break;
+                case "2":
+                    System.out.println("Type of task: ");
+                    String taskType = in.nextLine();
+                    System.out.println("Description of task: ");
+                    String taskDescription = in.nextLine();
+                    switch (taskType) {
+                        case "T":
```

Comment:
Should unindent all your case statements, as suggested in the coding standard provided